src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    // Length of horizontal line
+    private static final int HORIZONTAL_LINE_LENGTH = 80;
+
+    public static void printHorizontalLine() {
+        for (int i = 0; i < HORIZONTAL_LINE_LENGTH; i++) {
+            System.out.print("\u2500");
+        }
+
+        System.out.println();
+    }
+
+    public static void main(String[] args) throws IOException {
+        Scanner sc = new Scanner(System.in);
+        List<Task> tasks = Save.loadTaskList();
+
+        // Greeting
+        printHorizontalLine();
+        System.out.println(
+            "Karen:\n" +
+            "Can I speak to your manager?\n" +
+            "Just kidding, this is Karen. How can I help you today?"
+        );
+        printHorizontalLine();
+
+        // Commands
+        String command = sc.nextLine();
+
+        while (!command.equals("bye")) {
+            printHorizontalLine();
+            System.out.println("Karen:");
+            try {
+                if (command.equals("list")) {
+                    for (int i = 0; i < tasks.size(); i++) {
+                        Task currTask = tasks.get(i);
+                        System.out.println((i + 1) + ". " + currTask.toString());
+                    }
+                } else if (command.startsWith("mark") || command.startsWith("unmark")) {
+                    int taskNumber = Integer.parseInt(command.split(" ")[1]);
+                    String action = command.split(" ")[0];
+                    Task currTask = tasks.get(taskNumber - 1);
+
+                    if (action.equals("mark")) {
+                        currTask.markAsDone();
+                        Save.saveTaskList(tasks);
+                        System.out.println(
+                            "Congrats, I guess you get a medal?\n" +
+                            currTask.toString()
+                        );
+                    } else if (action.equals("unmark")) {
+                        currTask.markAsNotDone();
+                        Save.saveTaskList(tasks);
+                        System.out.println(
+                            "Why are you so lazy?\n" +
+                            currTask.toString()
+                        );
+                    }
+                } else if (command.startsWith("delete")) {
+                    int taskNumber = Integer.parseInt(command.split(" ")[1]);
+                    Task taskToDelete = tasks.get(taskNumber - 1);
+                    tasks.remove(taskToDelete);
+                    Save.saveTaskList(tasks);
+
+                    System.out.println(
+                        "Okay okay, this has been removed:\n" +
+                        taskToDelete.toString() +
+                        "\nNow you have " + tasks.size() + " tasks left."
```

Comment:
Maybe you can consider to put the '+' operator for every line breaks in a new line? :thinking:
```suggestion
                        "Okay okay, this has been removed:\n" 
                        + taskToDelete.toString() 
                        + "\nNow you have " + tasks.size() + " tasks left."
```